app/code/core/Mage/Eav/Model/Entity/Increment/Numeric.php

```diff
@@ -45,8 +45,8 @@ public function getNextId()
 
         if (empty($last)) {
             $last = 0;
-        } else if (strpos($last, (string)$this->getPrefix()) === 0) {
-            $last = (int)substr($last, strlen($this->getPrefix()));
+        } else if (!empty($prefix = (string)$this->getPrefix()) && strpos($last, $prefix) === 0) {
+            $last = (int)substr($last, strlen($prefix);
```

Comment:
you deleted one `)` to much, and now it produces a syntax error

Reply:
Classic. Sorry about that. It's fixed.